src/ompl/tools/thunder/Thunder.h

```diff
@@ -183,16 +187,42 @@ namespace ompl
             /** \brief Allow accumlated experiences to be processed */
             bool doPostProcessing() override;
 
-            void addSolutionToQueue(const std::shared_ptr<ompl::geometric::PathGeometric>& path_ptr) {
-              queuedSolutionPaths_.push_back(*path_ptr); 
+            void addSolutionToQueue(const std::shared_ptr<ompl::geometric::PathGeometric> &path_ptr)
+            {
+                queuedSolutionPaths_.push_back(*path_ptr);
             }
```

Comment:
revert formatting changes. please use .clang-format

Reply:
I use the command clang-format -i Thunder.cc
Isn't this what we want?

Reply:
no, you have to use the `.clang-format` config